CHANGELOG.md

```diff
@@ -29,7 +29,7 @@ release.
   <th><a href="doc/changelogs/CHANGELOG_V9.md">9</a></th>
   <th title="LTS Until 2019-12"><a href="doc/changelogs/CHANGELOG_V8.md">8</a><sup>LTS</sup></th>
   <th title="LTS Until 2019-04"><a href="doc/changelogs/CHANGELOG_V6.md">6</a><sup>LTS</sup></th>
-  <th title="LTS Until 2018-04"><a href="doc/changelogs/CHANGELOG_V4.md">4</a><sup>LTS</sup></th>
+  <th title="Unsupported Since 2018-05-01"><a href="doc/changelogs/CHANGELOG_V4.md">4</a><sup>EOL</sup></th>
```

Comment:
Maybe `End-of-life since 2018-05-01`?

Reply:
I copied this from the [previous text for v0.10/v0.12](https://github.com/nodejs/node/blame/effeff1843db3e7e47cf0242cd25273e083b2180/CHANGELOG.md#L26-L27), but I'm fine with changing it. What do you think?

Reply:
Ah didn't spot that. I'd be fine with either.

Reply:
I prefer `End-of-life since 2018-05-01` but it is also ok this way.

Reply:
I updated the message to `End-of-life since 2018-05-01`.